tests/vspec/test_overlay_on_instance/overlay_2.vspec

```diff
@@ -50,22 +50,6 @@ A.B.Row3.Right.C:
   datatype: int8
   my_id: Lund
 
-# Adding one outside instance range, shall not inherit original comment
```

Comment:
Should be OK to change behavior when using `Row5` like below, I could anyway not find that we have actually specified in documentation what shall happen. But I am thinking if we should remove the Row5 example, or rather keep it with a note saying that we do not do any "smart" name analysis and instead update expected.json

But I do not really have a strong opinion!

Reply:
I will revert it and comment it